mailservice-api/src/main/java/ca/bc/gov/ag/mail/MailController.java

```diff
@@ -49,23 +40,10 @@ public void sendMessage(@RequestBody @Valid MailMessage mailMessage) throws URIS
         logger.debug("Request to /sendMessage, uuid: {}", mailMessage.getUuid());
 
         try {
-            List<String> attachments = ListUtils.emptyIfNull(mailMessage.getAttachments());
-            URI[] attachmentRefs = new URI[attachments.size()];
-            for (int i = 0; i < attachmentRefs.length; i++) {
-                attachmentRefs[i] = new URI(attachments.get(i));
-            }
```

Comment:
I noticed this code didn't get copied to the service, is that intentional?

Reply:
Yup, good observation.  That code was originally copied from the BPEL project. If you notice, the attachmentRefs variable is not even used, so I dropped it completely ... considered dead code.